pkg/wfc3/lib/subtools.c

```diff
@@ -163,14 +158,12 @@ int Full2Sub(WF3Info *wf3, SingleGroup *x, SingleGroup *full, int dq, int sci, i
     sci boolean says to copy the science pixels*/
 
   if (!wf3->subarray){
-    sprintf(MsgText,"Original image is not a subarray, check image...");
-    trlmessage(MsgText);
+    trlerror("Original image is not a subarray, check image...");
     return(status = HEADER_PROBLEM);
   }
 
   if (wf3->detector != CCD_DETECTOR){
-    sprintf(MsgText,"Full2Sub only valid for UVIS subarrays");
-    trlmessage(MsgText);
+    trlmessage("Full2Sub only valid for UVIS subarrays");
```

Comment:
To be consistent with your other similar change above.
```suggestion
    trlwarn("Full2Sub only valid for UVIS subarrays");
```